### PR TITLE
add cloudsql HA

### PR DESCRIPTION
**Pull request by salrashid123**
update cloudSQL sample:

- use CloudSQL v2
- show adding master + N read-replicas
- show user API to remove root and also add a user

### CODE REVIEW

File: examples/v2/sqladmin/README.md [hunk 1]
```diff
@@ -4,16 +4,27 @@
...
+- [roles/cloudsql.admin](https://cloud.google.com/iam/docs/understanding-roles#sql_name_short_roles)  role
+to the service account DM uses (```projectNumber@cloudservices.gserviceaccount.com```)
+
 ## Deploy the template
 
 Use `sqladmin.yaml` to deploy this example template. When ready, deploy
 with the following command:
 
+```
     gcloud deployment-manager deployments create my-database --config sqladmin.yaml
```
Comment (krwenholz) [inline examples/v2/sqladmin/README.md]: If you're going to wrap it in triple ticks, I would remove the 4 preceding spaces.

File: examples/v2/sqladmin/jinja/sqladmin.yaml [hunk 2]
```diff
@@ -5,4 +5,10 @@ resources:
 - name: database
   type: sqladmin.jinja
   properties:
-    tier: D1
+    tier: db-n1-standard-1
+    region: us-central1
+    readReplicas: 1
+    replicationType: SYNCHRONOUS
+    username: alice
+    password: foo
```
Comment (krwenholz) [inline examples/v2/sqladmin/jinja/sqladmin.yaml]: Until we have secrets management (a few quarters out still), I don't think we should be encouraging the storage of passwords in our templates. Can you remove the user addition for now? (From both jinja and python.)

Comment (salrashid123) [reply examples/v2/sqladmin/jinja/sqladmin.yaml]: done

File: examples/v2/sqladmin/README.md (hunk 1 above)
Comment (salrashid123) [reply examples/v2/sqladmin/README.md]: done;

**Comment by krwenholz** [comment]
Thanks for the contribution @salrashid123

**Pull request merged**